index.bs

```diff
@@ -657,19 +657,13 @@ spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
   </xmp>
 
   <section algorithm="window-portalhost">
-    The <dfn attribute for="Window">portalHost</dfn> attribute *must* run the following steps:
+    The <dfn attribute for="Window">portalHost</dfn> attribute's getter *must* run the following steps:
 
-    1. Let |windowProxy| be [=this=]'s {{WindowProxy}} object.
+    1. Let |context| be [=this=] {{Window}} object's [=browsing context=].
```

Comment:
You can just say "this's" if you want. Now that it's formally defined we'll probably eventually update HTML to use it uniformly, instead of "this X object". (But, either is fine.)

Reply:
Okay, I'll say "this's". Also fixed the xref to "Window/browsing context" instead of the general definition of "browsing context".

Reply:
Doh, I'll try to check linking destinations better next time!